bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -131,6 +131,11 @@ spec:
               DNS.3 = $my_hostname.$svc.$MY_POD_NAMESPACE.svc.cluster.local
               DNS.4 = localhost
               DNS.5 = 127.0.0.1
+              {{- if .Values.externalAccess.service.loadBalancerIPs }}
```

Comment:
I like this solution, but this would imply to update the chart with the requested ips to work and wait until the loadBalancerIPs are set to generate the certs, right?

Reply:
I'm not sure I fully understood you, but the creation of the certificates is independent of the creation of the LoadBalancer services, as the LoadBalancers are created with pre-defined static IPs. One would just as usual wait for both the Statefulset (thus Pods) and the LoadBalancer services to be running with the latter having the appointed static IP addresses as external IP addresses.

Reply:
Yes, that is what I meant, this solution is assuming that we have static IPs, however, this could not happen always, right?

We need to provide a way where we can create this having dynamically assigned IPs. Normally, LoadBalancers can be created to supply dynamic IPs. 

We always try to fix these kinds of issues by doing a two steps installation: [see this](https://github.com/bitnami/charts/blob/master/bitnami/opencart/templates/NOTES.txt)

Reply:
You are right, not always one has/can have/want to have static IPS.  When that's the case, we go with the two-step deployment process you referenced above.

On the other hand, when the static IPs are available*, we should make use of them and simplify the process (to only one step). As in this PR, only a one if-statement and appending the IPs to the cert SAN would be all needed. Luckily the title of this PR is very specific to this case, so it can be implemented then another PR could be made (by someone, who is not me :)) to complement the other situation.

*In fact, in corporations where certificates need to be signed by the internal CA, there must be previously reserved static IP addresses anyway.

Reply:
I agree with you @mnmami . We can't assume that static IPS will be used, but when they're used, we should them.